### ISSUE TITLE
move2: Processing and Analysing Animal Trajectories

### ISSUE THREAD

**Issue by basille** [cran-task-views/Tracking#70]
### Are you the author of the package?

No

### Is the package on a specific repository?

CRAN

### Please provide the URL of the package repository

https://cran.r-project.org/package=move2

### Is there a dedicated website for the documentation (e.g. a pkgdown website)?

https://bartk.gitlab.io/move2/

### If you are the author of the package, does it pass CRAN check tests?

Yes

### Additional information

`move2` provides tools to handle, manipulate and explore trajectory data, with an emphasis on data from tracked animals. It does so by providing the 'move2' class. `move2` also facilitates import of animal movement data from movebank. (**note that `move2` is a replacement of `move`, which is already listed in the Tracking CTV**)

Not exactly sure where to put it. It does provide some functions for "Track description", but most of the actual work (cleaning, filtering) is done through other utilities `dplyr`, `sf`). The core of the package is really the 'move2' class (basically same objective as `sftrack`). Maybe we should add a specific section in the CTV dedicated to classes for tracking data. This is central to our aim, but not really present in the CTV… In a sense, this is neither "Pre-processing" (we already have tracking data, not in a formal class though) nor "Post-processing" (converting tabular data into a `move2` class does not process much, it organises the data and turn it into a coherent object).

**Comment by basille** [comment]
@rociojoo what do you think?

**Comment by bart1** [comment]
Thanks for the suggestion to include it. It indeed provides base functionality that helps preprocessing while maintaining track level data, the goal is to facilitate processing this way. It also simplifies import from movebank. Let me know if there are any questions

**Comment by basille** [comment]
Getting back to this. Thanks @bart1 for your explanation.

@rociojoo I would still suggest to add a subsection for classes dedicated to tracking data (`sftrack`, `adehabitatLT`, `move2`…). I would actually create an entire new section "Track management", combining "Dedicated classes", "Visualization" and "Track description". Any objection?

**Comment by rociojoo** [comment]
Thank you @bart1 . 

@basille I'm not sure about creating a new class. The purpose of the CTV is to guide users for the types of tasks they want to do. Do you see users looking into a class category? What for? That should guide our decision and the scope of that section. 

Otherwise, based on your arguments and @bart1 's, this sounds like post-processing in the sense it makes a smooth transition from movebank to R. But I haven't played with it yet.

**Comment by basille** [comment]
@rociojoo I respectfully disagree here. The CTV is not explicit about its target, but it does follow the JAE review, which is explicit:

> This review is aimed at movement ecologists, whether they are potential users or developers of r packages. This study aimed to provide users with criteria through which they can select packages for specific analyses, and offers developers recommendations to maximize the utility of packages and strengthen the links within the r community.

As a consequence, highlighting foundational packages that offer the infrastructure for the whole ecosystem (i.e. formal classes) should be an objective, AFAIC. Besides, it is even a plus for users directly: I often find packages that do not rely on formal classes to be annoying to use.

This said, we should have this discussion in the [dedicated issue](https://github.com/cran-task-views/Tracking/issues/81) I opened for this.

**Comment by bart1** [comment]
To add to @basille , one of the goals of providing easy access to movebank data and providing a formal class that retains meta data (units, projections, timezones, track level information) is to facilitate different analysis without having to convert data in between while retaining data integrity. Now `wildlifeDI` and `recurse` can be used directly.

**Comment by basille** [comment]
Added to `[dev] Tracking.md`, will be included in the next checked version.

Specifically, added to:

- The new category `Formal classes` (see the [other thread](https://github.com/cran-task-views/Tracking/issues/81) for more details)
- `Post-processing` > `Import online data` (new sub-category).
